walkthroughs/howto-k8s-cross-cluster/README.md

```diff
@@ -20,7 +20,9 @@ Front app acts as a gateway that makes remote calls to colorapp. Front app has s
 
```

Comment:
Not introduced by PR but the region in cluster create (us-east-1) vs AWS_DEFAULT_region (us-west-2) are different. Can we please use just one